gammapy/datasets/map.py

```diff
@@ -393,7 +393,7 @@ def npred(self):
 
         if self.background:
             npred_total += self.npred_background()
-
+        npred_total.data[npred_total.data < 0.0] = 0
```

Comment:
Is the same command fine? Does `np.clip` provide better performance?

Reply:
I think @luca-giunti checked the implementation at the time and the "normal" masking was faster, if I remember correctly.